pkg/controllers/management/auth/user_attribute_handler.go

```diff
@@ -28,6 +28,10 @@ func (ua *UserAttributeController) sync(key string, obj *v3.UserAttribute) (runt
 		return nil, nil
 	}
 
+	if obj.ExtraByProvider == nil {
+		obj.NeedsRefresh = true
+	}
```

Comment:
I assume `obj.NeedsRefresh` will be set to false by the refresher daemon?

Reply:
Yes it will https://github.com/rancher/rancher/blob/56b57162f7941f811969beea6c27ae5317aafdf9/pkg/auth/providerrefresh/daemon.go#L89